app/main.py

```diff
@@ -2,0 +1,16 @@
+from app.cafe import Cafe
+from app.errors import NotWearingMaskError, VaccineError
+
+
+def go_to_cafe(friends: list[dict], cafe: Cafe) -> str:
+    need_mask = 0
+    for friend in friends:
+        try:
+            cafe.visit_cafe(friend)
+        except NotWearingMaskError:
+            need_mask += 1
+        except VaccineError:
+            return "All friends should be vaccinated"
+    if need_mask:
```

Comment:
Code Efficiency Issue: There is no need to compare `need_mask` with `0`. You can directly use `if need_mask:` which is more pythonic and efficient.